useradm/rbconfig.py

```diff
@@ -115,26 +115,26 @@
 # Valid account usertypes and descriptions.
 #
 usertypes = {
-	'founders':	'RedBrick founder',
-	'member':	'Normal member',
-	'associat':	'Graduate/associate member',
-	'staff':	'DCU staff member',
-	'society':	'DCU society',
-	'club':		'DCU club',
-	'projects':	'RedBrick/DCU/Course project account',
-	'guest':	'Guest account',
-	'intersoc':	'Account for society from another college',
-	'committe':	'Committee member or a position account',
-	'redbrick':	'RedBrick related account',
-	'dcu':		'DCU related account'
+        'founders':     'RedBrick founder',
+        'member':       'Normal member',
+        'associat':     'Graduate/associate member',
+        'staff':        'DCU staff member',
+        'society':      'DCU society',
+        'club':         'DCU club',
+        'projects':     'RedBrick/DCU/Course project account',
+        'guest':        'Guest account',
+        'intersoc':     'Account for society from another college',
+        'committe':     'Committee member or a position account',
```

Comment:
Not exactly part of what this change is about, but should it not be `committee`?